helm-charts/chatqna/Chart.yaml

```diff
@@ -9,9 +9,27 @@ dependencies:
   - name: tgi
     version: 0.9.0
```

Comment:
The latest version is updated to 1.0.0, for all components.

Reply:
sure. updated all the versions.